html/html-and-the-web_personal-website/index.html

```diff
@@ -97,6 +97,10 @@ <h2>Get in Touch</h2>
       </section>
     </main>
     <footer class="footer-container">
+<<<<<<< HEAD
+=======
+      <section></section>
+>>>>>>> 6628cfc (untracked file portfolio)
```

Comment:
Woops, looks like you've committed some markup directly from within a merge conflict. Might be worth deleting this bit of code altogether it looks like : )

Reply:
About this I want to discuss it with you. 
Also, How may I test it?